scripts/main.js

```diff
@@ -1,3 +1,54 @@
+//index.html
+//display the creat app element, when a user click the new app button
+var newAppBtn = document.querySelector("#new-app-btn");
+if (newAppBtn) {
+	newAppBtn.addEventListener("click", function(){
+		document.querySelector("#create-app").style.display = "block";
+		//create an new app, when a user click 'create button'
+		var createBtn = document.querySelector("#choose-create");
+		debugger;
+		if(createBtn){
+			createBtn.addEventListener("click",function(){
+				var appname = document.getElementsByTagName("input")[0].value;
+				console.log('appname_value: '+ appname);
+				//save the appname to the local storage
+				if(appname){
+					console.log('appname_localstorage: '+ appname);
+					localStorage.setItem("app-name", appname);
+					//redirect from index page to detail_app_page
+					//JS
+					//window.location.replace("detail_app_page.html");
+					//Jquery
+					 $(document).ready(function(){
+					 	var url = "detail_app_page.html";
+					 	$(location).attr('href',url);
+					 });
```

Comment:
You only need `$(document).ready(function(){})` if you want to define code that shouldn’t run before all HTML is ready. In your case, the HTML is already loaded and rendered, so you should be able to do just

```js
var url = "detail_app_page.html";
$(location).attr('href',url);
```

without the `$(document).ready(function(){` bit